dev/docker-compose.yml

```diff
@@ -51,8 +54,6 @@ services:
 
   client:
     image: conjurinc/cli5
-    # entrypoint: sleep
-    # command: infinity
```

Comment:
I'd be inclined to leave these in, and maybe even uncomment them. Because we no longer install the CLI in the conjur container, you almost always want to bring up a client.

Reply:
I'll leave them in. I also added a CLI dev tool to handle loading policy: https://github.com/cyberark/conjur/pull/529